prisma/language-tools
# feat(mongodb): completion for composite types

Closes https://github.com/prisma/language-tools/issues/1102

To test read instructions at 
https://github.com/prisma/language-tools/blob/joel/composite-types-completion/packages/vscode/CONTRIBUTING.md#pull-requests

Example schema to try, copy and paste then save into a schema.prisma file:
```prisma
datasource db {
        provider = "mongodb"
        url      = env("DATABASE_URL")
}
type Address {
        street String
        number Int
        alpha  Alpha
}
type Alpha {
        bravo  Bravo
        helloA Int
}
type Bravo {
        something  String
        helloBravo Int
}
model User {
        id      String  @id @default(auto()) @map("_id") @db.ObjectId
        email   String
        address Address

        // should autocomplete, example of end result:
        // @@index([email, address.alpha.bravo.helloBravo])
        
        // try autocompletion by adding a `, address.` below ⬇️ and hitting `Ctrl + space` keys
        @@index([email])
}
```
